app/services/mediaflux_script_factory.rb

```diff
@@ -89,7 +81,6 @@ def build_create_script
             :StorageCapacity < :Size "#{project.metadata_json['storage_capacity']['size']['requested']}>" :Unit #{project.metadata_json['storage_capacity']['unit']['requested']}"
             :StoragePerformance "#{project.metadata_json['storage_performance_expectations']['requested']}"
             :ProjectPurpose "#{project.metadata_json['project_purpose']}"
-            :Submission < :RequestedBy "#{requested_by}" :RequestDateTime "#{requested_date}" >
```

Comment:
Why remove the Submission from the mediaflux object?

Reply:
Restored it